src/helperFunctions.js

```diff
@@ -0,0 +1,7 @@
+export function getErrorClass(error) {
```

Comment:
please put this under components/errorhelpers or something. not in src.

Reply:
I found one on reddit said that we can put our helpers in a folder called utils so I put it in that folder in src. I wish that is fine.

Reply:
what are your teammates doing? most important thing is consistency in codebase, if no one else has helpers you can src/utils/errorHelpers

helperfunctions too general a name. adn dont pollute src/ top level